biztalk/core/gmsa-config.md

```diff
@@ -0,0 +1,84 @@
+---
+title: BizTalk Server - group managed service account (gMSA) | Microsoft Docs
+description: Configure and use BizTalk Server with group managed service account (gMSA)
+author: msjaydeep
+ms.author: jaah
+manager: dougeby
+ms.date: 01/03/2019
+ms.topic: config
+ms.prod: biztalk-server
+
+# optional metadata
+
+#ROBOTS:
+
+ms.reviewer: 
+ms.suite:
+ms.tgt_pltfrm:
+ms.assetid: 
+ms.custom: 
+---
+
+# Using Group Managed Service Account for BizTalk Server Features
+
+BizTalk 2020 Supports Group Managed Service Account. Detailed information about gMSA can be found [here](https://docs.microsoft.com/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) .
+
+With gMSA support users can continue to run BizTalk service without having the need to change passwords.
+
+There are multiple BizTalk Features and following table shows the features where gMSA is supported
+
+| Feature Name | Supported |
+| --- | --- | 
+| Enterprise SSO | No |
+| Group | N/A |
+| BizTalk Runtime | Yes |
+| Business Rules Engine | Yes |
+| BAM Tools | Yes (for Bam Alerts) |
+| BAM Portal | Only for Application Pool Account |
+| BizTalk EDI/AS2 Runtime | N/A |
+| Rest API | Yes |
+| BizTalk TMS | Yes |
+
+New installations of BizTalk may be configured with gMSA(where ever supported) by running Microsoft BizTalk Server Configuration. 
+
+> [!NOTE]
+> gMSA is not available in basic configuration.
+
+The features where gMSA is supported shows a check box which when checked password gets disabled. The user name should be set to appropriate gMSA account.
+
```

Comment:
images do not have $ in the gmsa account, this is misleading.

Reply:
done